db/memtable.cc

```diff
@@ -81,7 +84,8 @@ Iterator* MemTable::NewIterator() {
 
 void MemTable::Add(SequenceNumber s, ValueType type,
                    const Slice& key,
-                   const Slice& value) {
+                   const Slice& value,
+                   const ExpiryTime & expiry) {
   // Format of an entry is concatenation of:
```

Comment:
Would be good to update this comment to match what is actually put into the buffer.